sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -684,7 +685,10 @@ object DateTimeUtils {
       case "TH" | "THU" | "THURSDAY" => THURSDAY
       case "FR" | "FRI" | "FRIDAY" => FRIDAY
       case "SA" | "SAT" | "SATURDAY" => SATURDAY
-      case _ => -1
+      case _ =>
+        val invalidInput = string.replace(UTF8String.fromString("""""""),
```

Comment:
I think `"\""` is more readable than `"""""""`

Reply:
Finally I think we can use the input directly